app/services/data/get-individual-claim-details.js

```diff
@@ -144,6 +144,14 @@ function getClaimExpenses (claimId) {
 function getClaimDeductions (claimId) {
   return knex('ClaimDeduction')
     .where({'ClaimId': claimId, 'IsEnabled': true})
+
+  .then(function (claimDeductions) {
+    claimDeductions.forEach(function (claimDeductions) {
+      claimDeductions.Amount = Number(claimDeductions.Amount).toFixed(2)
+    })
+
+    return claimDeductions
+  })
```

Comment:
Can you move this `.then` block up a line and indent it by 2 spaces so that it's in line with the `.where` statement above?